schema-registry/common/src/main/java/com/hortonworks/registries/schemaregistry/errors/InvalidSchemaBranchDeletionException.java

```diff
@@ -25,4 +29,14 @@ public InvalidSchemaBranchDeletionException(String message) {
     public InvalidSchemaBranchDeletionException(String message, Throwable throwable) {
         super(message, throwable);
     }
+
+    public static InvalidSchemaBranchDeletionException getSchemaVersionTiedToOtherBranchException(Map<Integer, List<String>> versionToBranchMap) {
+        StringBuffer message = new StringBuffer();
```

Comment:
You may want to use `StringBuilder` instead of `StringBuffer` as no need to have synchronized charsequence here.